src/index.html

```diff
@@ -5,10 +5,48 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
-    <h1>Moyo header</h1>
+
+  <body id="home">
+    <header class="header">
+      <a class="logo__link" href="#home">
+        <img class="logo_img" src="images/logo.png"
+          alt="images_logo">
+      </a>
+      <nav class="nav">
+        <ul class="nav__links">
+          <li class="nav___item first-item">
```

Comment:
u need only 2 underlines

```suggestion
          <li class="nav__item first-item">
```

Reply:
Also, u can use Pseudo-classes [LINK](https://developer.mozilla.org/en-US/docs/Web/CSS/Pseudo-classes) instead `first-item` class
Like, `:first-child`, `:last-child` etc